pkg/bootstrap/init.go

```diff
@@ -251,51 +200,33 @@ func yamlToObject(yamlContent []byte) (*unstructured.Unstructured, error) {
 	return obj, nil
 }
 
-func createOrUpdateResourcesFromDir(dir string, client client.Client, reader client.Reader) error {
-	files, err := ioutil.ReadDir(dir)
+func createOrUpdateResourcesFromAnnotation(client client.Client, reader client.Reader) error {
```

Comment:
I'd like to separate the CS resources and extra resources into two functions as the behavior before.

And you need to add wait for on the CS resources creation.

Reply:
done